Artifacts/windows-slack/Artifactfile.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Slack",
+  "description": "Installs Slack using the Chocolatey package manager",
+  "tags": [
+    "Windows",
+    "Slack"
+  ],
+  "iconUri": "https://chocolatey.org/content/packageimages/slack.1.0.0.png",
+  "targetOsType": "Windows",
+   "parameters": {
+        "installUsername": {
```

Comment:
I appreciate that, but users won't see the readme or a link to it when they use the Azure portal UI to create VMs and add artifacts, so having it clearer in the parameter name and description will make a big difference.

Still, I think the phrase "credentials of user for hich you want to install the package" isn't necessarily clear between local VM account and Slack account. I think you mean the former, right?